pkg/inject/inject.go

```diff
@@ -422,35 +426,39 @@ func (conf *ResourceConfig) injectPodSpec(patch *Patch) {
 		}
 	}
 
-	if conf.globalConfig.GetIdentityContext() != nil {
-		log.Warn("TLS configuration exists but is currently not available")
+	sidecar.Env = append(sidecar.Env, v1.EnvVar{
+		Name:  "LINKERD2_PROXY_IDENTITY_DISABLED",
+		Value: "Identity configuration is unavailable",
+	})
+	if idctx := conf.globalConfig.GetIdentityContext(); idctx != nil {
+		log.Warn("Ignoring Identity configuration.")
```

Comment:
Curious as to what will this do eventually? I assume it won't stay as `log.Warn()`.

Reply:
https://github.com/linkerd/linkerd2/blob/ver/identikit/pkg/inject/inject.go#L484-L554